pydantic/main.py

```diff
@@ -170,8 +161,7 @@ class BaseModel(_repr.Representation, metaclass=ModelMetaclass):
         __pydantic_validator__: typing.ClassVar[SchemaValidator]
         __pydantic_core_schema__: typing.ClassVar[CoreSchema]
         __pydantic_serializer__: typing.ClassVar[SchemaSerializer]
-        __pydantic_validator_functions__: typing.ClassVar[_decorators.ValidationFunctions]
-        __pydantic_serializer_functions__: typing.ClassVar[_decorators.SerializationFunctions]
+        __pydantic_decorators__: typing.ClassVar[_decorators.DecoratorInfos]
```

Comment:
I think it would be helpful to add a comment indicating that this thing will hold information about validators and serializers (in addition to the docstrings I mentioned elsewhere)